### PR TITLE
[kube-prometheus-stack]  Add support for templating in excludedFromEnforcement parameter

### PR DESCRIPTION
**Pull request by amuraru**
#### What this PR does / why we need it

Support for sensible defaults in downstream charts.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by Allex1** [comment]
@monotek @andrewgkew @desaintmartin @gianrubio @gkarthiks @scottrigby @Xtigyro please have a look

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.11.0
+version: 39.11.1
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:25]:
```suggestion
version: 39.13.2
```

please merge after: https://github.com/prometheus-community/helm-charts/pull/2430#pullrequestreview-1105487003 which has version 39.13.1

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 1 above)
Comment (amuraru) [reply charts/kube-prometheus-stack/Chart.yaml]: Done

File: charts/kube-prometheus-stack/Chart.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.13.0
+version: 39.13.2
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:25]:
```suggestion
version: 39.13.3
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — APPROVED** [review]

**Pull request merged**